std_msgs/Header header
nav_msgs/MapMetaData info
int16[] data
